test/e2e/upgrade_from_latest_test.go

```diff
@@ -24,6 +24,35 @@ func runUpgradeFromReleaseFlow(test *framework.ClusterE2ETest, latestRelease *re
 	test.DeleteCluster()
 }
 
+func runMulticlusterUpgradeFromReleaseFlowAPI(test *framework.MulticlusterE2ETest, latestRelease *releasev1.EksARelease, wantVersion anywherev1.KubernetesVersion, opts ...framework.ClusterE2ETestOpt) {
+	test.CreateManagementClusterForVersion(latestRelease.Version, framework.ExecuteWithEksaRelease(latestRelease))
+	test.RunConcurrentlyInWorkloadClusters(func(wc *framework.WorkloadCluster) {
+		wc.ApplyClusterManifest()
+		wc.WaitForKubeconfig()
+		wc.ValidateClusterState()
+	})
+	// Adding this manual wait because old versions of the cli don't wait long enough
+	// after creation, which makes the upgrade preflight validations fail
+	test.ManagementCluster.WaitForControlPlaneReady()
```

Comment:
Do we still need this?

Reply:
I don't think so. And if we do, it should go before creating the workload clusters

Reply:
Ah, let me double check. At some point I was experiencing upgrade preflight validations failure, so I had decided to include this.

Reply:
Not needed after all, will remove.